ir/constant/const_int.go

```diff
@@ -127,5 +127,118 @@ func (c *Int) Ident() string {
 			panic(fmt.Errorf("invalid integer value of boolean type; expected 0 or 1, got %d", x))
 		}
 	}
+	// Output x in hexadecimal notation if x is positive, greater than or equal
+	// to 0x1000 and has a significantly lower entropy than decimal notation.
+	const minEntropyDiff = 0.3
```

Comment:
maybe move to the global level?

Reply:
> maybe move to the global level?

To have both metrics in one place, I'd prefer to keep `minEntropyDiff` defined alongside of `threshold`.

```go
	const minEntropyDiff = 0.3
	threshold := big.NewInt(0x1000) // 4096
```

Since `minEntropyDiff` is only used in `constant.Int.Ident` I think we can keep it locally scoped to the `Ident` method. If we ever were to re-use the `minEntropyDiff` outside of `Int.Ident`, then lets move it to global scope.